Jellyfin.Server/Program.cs

```diff
@@ -187,6 +187,30 @@ private static ServerApplicationPaths CreateApplicationPaths(StartupOptions opti
                 Directory.CreateDirectory(configDir);
             }
 
+            string cacheDir = Environment.GetEnvironmentVariable("JELLYFIN_CACHE_DIR");
+            if (string.IsNullOrEmpty(cacheDir))
+            {
+                if (options.ContainsOption("-cachedir"))
+                {
+                    cacheDir = options.GetOption("-cachedir");
+                }
+                else if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+                {
+                    // $XDG_CACHE_HOME defines the base directory relative to which user specific non-essential data files should be stored.
+                    cacheDir = Environment.GetEnvironmentVariable("XDG_CACHE_HOME");
```

Comment:
Why no 'jellyfin' folder under XDG_CACHE_HOME?

Reply:
There is.

Reply:
Only if XDG_CACHE_HOME is null do we drop to using HOME/.cache/ and append the 'jellyfin'.  If XDG_CACHE_HOME is valid, then no 'jellyfin' is not appended.